runtime/io.c

```diff
@@ -247,11 +247,19 @@ CAMLexport int caml_channel_binary_mode(struct channel *channel)
    end of the flush, or false if some data remains in the buffer.
  */
 
-CAMLexport int caml_flush_partial(struct channel *channel)
+static int flush_partial(struct channel *channel, bool exn_if_closed)
```

Comment:
```suggestion
static bool flush_partial(struct channel *channel, bool exn_if_closed)
```

might as well make that a `bool`